### PR TITLE
fix(deps): update undici to 4.4.6

### PR DESCRIPTION
**Pull request by janpio**
test https://github.com/nodejs/undici/pull/989

### CODE REVIEW

File: packages/engine-core/package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "new-github-issue-url": "0.2.1",
     "p-retry": "4.6.1",
     "terminal-link": "2.1.1",
-    "undici": "^4.0.0"
+    "undici": "github:nodejs/undici#lzy-llhttp"
```
Comment (Jolg42) [inline packages/engine-core/package.json:58]: We cannot merge this, publish in Buildkite won't take it for publishing

**Review by Jolg42 — CHANGES_REQUESTED** [review]
We need to either publish the package temporarily somewhere on npm or wait for a new undici release.

**Comment by ronag** [comment]
4.4.6 published

**Comment by Jolg42** [comment]
And it works! Thank you @ronag!

**Pull request merged**